### PR TITLE
Move AppEntity, AppInitialization, AppInstallDialog, Authentication, Call, Location and Logs capabilities to new ApiInput components

### PR DESCRIPTION
**Pull request by MarcinJuraszek**

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : f4f5bf329c24c2d465750253c5a2b6144a2bdf13</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 256a27a83154f07d1a6087f8f5959ce19d4943f9</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : 218a9760c00ee78452ad45c688b3b722d50cdec2</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 256a27a83154f07d1a6087f8f5959ce19d4943f9</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : 910ac96eb3c47f891310b2e942d8a95451f4157b</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.33 KB</td><td>57.32 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 13d5e381f8c328eca83c944965e4a50132b2ad58</p>

**Review by aeunms — APPROVED** [review]
Super thorough and clean, thank you!

**Comment by TrevorJoelHarris** [comment]
:fire: 

---
In reply to: [972123109](https://github.com/OfficeDev/microsoft-teams-library-js/pull/722#issuecomment-972123109) [](http://example.com/codeflow?ancestors=972123109,972123109)

**Comment by erinha** [comment]
<p>Analyzed commit id : 7e14e07c0ff766cd64fb0a40f656b7478ca39294</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 6b62e4aa75c6a921eca3f576d9cee06ed0a4f16e</p>

File: apps/teams-test-app/src/components/AppInitialization.tsx [hunk 1]
```diff
@@ -2,22 +2,55 @@ import { app } from '@microsoft/teams-js';
...
+    },
+  });
 
-  const notifySuccess = (): void => {
-    app.notifySuccess();
-    setNotifySuccessRes('called');
-  };
+const NotifyFailure = (): React.ReactElement =>
+  ApiWithTextInput<app.FailedReason>({
+    name: 'appInitializationFailure2',
```
Comment (TrevorJoelHarris) [inline apps/teams-test-app/src/components/AppInitialization.tsx:29]: >2 [](http://example.com/codeflow?start=35&length=1)

what happened to 1

Comment (MarcinJuraszek) [reply apps/teams-test-app/src/components/AppInitialization.tsx:29]: There isn't "1" per se, we do have `appInitializationFailure` already, which has to stay for now before I can move the E2E scenario tests over to use the new one. So I went with `2` for these new items. Do you have a better suggestion on how to do these N/N-1 cases where tests have to be updated as well?

Comment (TrevorJoelHarris) [reply apps/teams-test-app/src/components/AppInitialization.tsx:29]: Oh! There is it is. No, I think 2 is fine. Just missed the BoxAndButton at the bottom

**Review by TrevorJoelHarris — APPROVED** [review]
:shipit:

**Pull request merged**